### PR TITLE
Some slight grammar fixes in JSON Parser and Printer

### PR DESCRIPTION
**Pull request by moxie-coder**
These seem needed, any feedback is appreciated 👍

### CODE REVIEW

File: std/haxe/format/JsonParser.hx [hunk 1]
```diff
@@ -23,7 +23,7 @@
 package haxe.format;
 
 /**
-	An implementation of JSON parser in Haxe.
+	An implementation of the JSON parser in Haxe.
```
Comment (kLabz) [inline std/haxe/format/JsonParser.hx]: I'm really not sure about those.

Comment (moxie-coder) [reply std/haxe/format/JsonParser.hx]: I'm honestly not sure either, I thought it felt appropriate

Comment (Aurel300) [reply std/haxe/format/JsonParser.hx]:
```suggestion
	An implementation of a JSON parser in Haxe.
```

File: std/haxe/format/JsonParser.hx [hunk 2]
```diff
@@ -37,7 +37,7 @@ class JsonParser {
 		JSON objects are parsed into anonymous structures and JSON arrays
 		are parsed into `Array<Dynamic>`.
 
-		If given `str` is not valid JSON, an exception will be thrown.
+		If given `str` is not a valid JSON, an exception will be thrown.
```
Comment (kLabz) [inline std/haxe/format/JsonParser.hx]: That one seems worse than original wording to me.

Comment (moxie-coder) [reply std/haxe/format/JsonParser.hx]: it didn't seem right to me compared to now to be honest

Comment (kLabz) [reply std/haxe/format/JsonParser.hx]: For example, `var foo = false;` is not "a valid Haxe", it's "valid Haxe".

Comment (moxie-coder) [reply std/haxe/format/JsonParser.hx]: fair enough

**Comment by kLabz** [comment]
- [ ] Also, CI failures _are_ related. If you change documentation for core types, you have to update documentation on target specific versions too.

**Comment by moxie-coder** [comment]
> Also, CI failures _are_ related. If you change documentation for core types, you have to update documentation on target specific versions too.

okay, good to know

**Comment by Simn** [comment]
Thanks!

**Comment by moxie-coder** [comment]
you're welcome

**Comment by kLabz** [comment]
@Simn But the PR was not ready, it breaks API generation

**Comment by kLabz** [comment]
@MemeHovy reverted your PR. Feel free to open a PR again while making sure it's complete this time.

**Comment by Simn** [comment]
Huh, I could swear I looked at it and it was green... Sorry about that!

**Pull request merged**